virttest/qemu_storage.py

```diff
@@ -1096,6 +1096,129 @@ def snapshot_apply(self):
+            cmd += " bitmap --add -f %s %s %s" % (
+                self.image_format, self.image_filename, bitmap_name)
+            if granularity:
+                cmd += ' -g %s' % granularity
+        else:
+            raise exceptions.TestError(
+                "Image format %s is not supported, check please." %
+                self.image_format)
+
+        process.run(cmd)
+
+    def bitmap_remove(self, bitmap_name):
+        """
+        Remove a bitmap from a image.
+
+        :param bitmap_name: The name of the bitmap
+        """
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --remove -f %s %s %s" % (
+                self.image_format, self.image_filename, bitmap_name)
+        else:
+            raise exceptions.TestError(
+                "Image format %s is not supported, check please." %
+                self.image_format)
+
+        process.run(cmd)
+
+    def bitmap_clear(self, bitmap_name):
+        """
+        Clear a bitmap of a image.
+
+        :param bitmap_name: The name of the bitmap
+        """
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --clear -f %s %s %s" % (
+                self.image_format, self.image_filename, bitmap_name)
+        else:
+            raise exceptions.TestError(
+                "Image format %s is not supported, check please." %
+                self.image_format)
+
+        process.run(cmd)
+
+    def bitmap_enable(self, bitmap_name):
+        """
+        Enable a bitmap of a image.
+
+        :param bitmap_name: The name of the bitmap
+        """
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --enable -f %s %s %s" % (
+                self.image_format, self.image_filename, bitmap_name)
+        else:
+            raise exceptions.TestError(
+                "Image format %s is not supported, check please." %
+                self.image_format)
+
+        process.run(cmd)
+
+    def bitmap_disable(self, bitmap_name):
+        """
+        Disable a bitmap of a image.
+
+        :param bitmap_name: The name of the bitmap
+        """
+
+        cmd = self.image_cmd
+        if self.image_format == 'qcow2':
+            cmd += " bitmap --disable -f %s %s %s" % (
+                self.image_format, self.image_filename, bitmap_name)
+        else:
+            raise exceptions.TestError(
+                "Image format %s is not supported, check please." %
+                self.image_format)
+
+        process.run(cmd)
+
+    def bitmap_merge(self, bitmap_name_source, bitmap_name, bitmap_source,
+                     params, root_dir):
+        """
+        Merge a bitmap from source image to target image.
+
+        :param bitmap_name_source: The name of the bitmap for source image
+        :param bitmap_name: The name of the bitmap for target image
+        :param bitmap_source: The tag for source image
```

Comment:
```suggestion
        :param bitmap_name_target: The name of the bitmap for target image
        :param source_image: The tag for source image
```
or 
```suggestion
        :param bitmap_name_target: The name of the bitmap for target image
        :param bitmap_image_source: The tag for source image
```

I think this named way is easy to distinguish with others arguments.
Since the previous name of arguments that make me confused although we have claim them by the docsting.
But the docstring of each arguments are still a little bit of mismatch.
Especially, between `bitmap_name_source` and `bitmap_source`.
How about your idea?
@zhencliu  and @hellohellenmao

Reply:
Make sense, either of yongxue's suggestions is OK for me.